docs/en/operations/table_engines/mergetree.md

```diff
@@ -302,6 +305,12 @@ SELECT count() FROM table WHERE u64 * i32 == 10 AND u64 * length(s) >= 1234
 
     The same as `ngrambf_v1`, but instead of ngrams stores tokens, which are sequences separated by non-alphanumeric characters.
 
+- `bloom_filter` — Stores [bloom filter](https://en.wikipedia.org/wiki/Bloom_filter) for the specified columns.
```

Comment:
It actually accepts an optional parameter: [false positives](http://pages.cs.wisc.edu/~cao/papers/summary-cache/node8.html)

Reply:
@zhang2014 Does this optional parameter is a probability of false positive response?

Reply:
Yes, it should be a float between 0 and 1.